src/less/src/common.less

```diff
@@ -79,6 +79,8 @@ input:not([type=button]):not([type=submit]).w2ui-input, textarea.w2ui-input {
     background-color: @input-background-color;
     line-height: normal;
     &:focus {
+        outline-style: solid;
+        outline-width: 1px;
```

Comment:
Why don't you like regular focus style. I think that shadow kind of style looks good. You are welcome to write a custom css style that you can load on top of regular.